--- /usr/lib/python3.9/site-packages/libqtile/drawer.py	2021-02-18 02:44:21.000000000 +0800
+++ drawer.py	2021-02-19 16:55:08.828800267 +0800
@@ -168,7 +168,8 @@
                 self.drawer.set_source_rgb(self.border_color)
 
             # change to only fill in bottom line
-            opts[1] = self.height - self.border_width  # y
+            # [rydesun]: top line
+            # opts[1] = self.height - self.border_width  # y
             opts[3] = self.border_width  # height
 
             self.drawer.fillrect(*opts)
